### ISSUE TITLE
Accessibility: "Skip to" links not visible

### ISSUE THREAD

**Issue by MomoMoses** [johnjung/bmrcportal#162]
On this page https://bmrc.lib.uchicago.edu/portal/help/
there are 3 links to skip content
1. skip to main content (which seems OK, even though I can't see it)
2. skip to sidebar
3. skip to footer

the last two aren't visible and they are supposed to be, I guess.

**Comment by johnjung** [comment]
Was this from Siteimprove? If so, I don't see this in their report- could you please copy and paste a link to this one? Alternatively, if this didn't come from Siteimprove, where did the recommendation to make the links visible come from?

**Comment by MomoMoses** [comment]
Yep definitely SiteImprove. Let me try to find it again. Also same issue on "Resources" page of main BMRC site.

**Comment by MomoMoses** [comment]
[site Improve link](https://my2.siteimprove.com/Inspector/1119201/A11Y/Page?pageId=34260554027&impmd=A7371DDD51435E8E1C2BE63D27B85ED4&expand=potential#/sia-r57/failed/YmlYbGY1VXM=;N2M4ZjM5N2Y=)

here's the error message text for more context:
Issue
Text not included in an ARIA landmark

**Comment by johnjung** [comment]
This is now fixed in https://github.com/uchicago-library/bmrc/commit/8af63b2599f32d39f33893c4409c6a321a12ae3d and live on bmrc.lib. Please confirm that this resolves the issue in Siteimprove, and if so, please feel free to close out this issue.

**Comment by MomoMoses** [comment]
SiteImprove gives it the thumbs up!
